docs/properties.md

```diff
@@ -27,13 +27,16 @@ Specifies OS disk characteristics.
     * `StandardSSD_LRS` - Standard SSD
     * `Standard_LRS` - Standard HDD
 
+Note that VM sizes ending in `s` support Premium SSD storage. Those that do not end in `s`, such as `Standard_DC8_v2` only support Standard storage.
+
 ### VM compute power
 Specifies VM compute characteristics
 
 * Path: `properties/vmProfiles[]/vmSize`
 * Values:
     * `Standard_DC2s`
     * `Standard_DC4s`
+    * `Standard_DC8_v2`
```

Comment:
Also, can you also add Standard_DC4s_v2, Standard_DC2s_v2 and Standard_DC1s_v2

Reply:
Done